backends/platform/sdl/sdl.h

```diff
@@ -163,9 +163,10 @@ class OSystem_SDL : public ModularMutexBackend, public ModularMixerBackend, publ
 	int _defaultGLMode;
 
 	/**
-	 * Creates the merged graphics modes list
+	 * Create and clear the merged graphics modes list
```

Comment:
Those comments are used for https://doxygen.scummvm.org and this change will document `setupGraphicsMode()` as `Create and clear the merged graphics modes list`, which is confusing. So this comment should be split in two.

I will make the change now, force push to your branch, and then merge the pull request.